mlem/contrib/flyio/meta.py

```diff
@@ -101,6 +101,7 @@ def _create_app(self, state: FlyioAppState):
                 "region": self.region
                 or self.get_env().region
                 or project_config("", section=FlyioConfig).region,
+                "no-deploy": True,
```

Comment:
What this "no-deploy" option does?

Reply:
By default this command will try to actually run something. But at this stage we only want to create app and not run it (we don't have docker dir yet)